### PR TITLE
Update CONTRIBUTING.md with coding standards and best practices

### PR DESCRIPTION
**Pull request by dermatz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -124,18 +121,30 @@ Use `${txtrst}` to reset the color back to default terminal color.
...
 ---
 
+## VSCODE USERS
+
+VSCode User automaticly use our VSCode Workspace Settings, located in `.vscode/settings.json`.
+We support [Git commit message helper](https://marketplace.visualstudio.com/items?itemName=D3skdev.git-commit-message-helper) to add
+correct commit message with prefixes based on your branch name given from github with issue-id.
+e.G. `#123 - Commit Message ...`
+
+![Demo Git commit message helper](https://github.com/d3skdev/git-prefix/raw/master/images/demo.gif)
```
Comment (Morgy93) [inline CONTRIBUTING.md:131]:
```suggestion
## VSCode Users

VSCode users automatically apply the workspace settings defined in `.vscode/settings.json`. 

To streamline your workflow, we recommend using the [Git Commit Message Helper](https://marketplace.visualstudio.com/items?itemName=D3skdev.git-commit-message-helper) extension. This tool ensures your commit messages are formatted correctly with the appropriate prefixes, based on the branch name and issue ID from GitHub.

For example: `#123 - Commit Message ...`

![Git Commit Message Helper Demo](https://github.com/d3skdev/git-prefix/raw/master/images/demo.gif)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -124,18 +121,30 @@ Use `${txtrst}` to reset the color back to default terminal color.
...
+
+---
+
 ## Best Practices
 
 -   Write clear and concise comments
--   Avoid global variables
+-   Avoid global variables, use `local VARIABLE` if possible
```
Comment (Morgy93) [inline CONTRIBUTING.md:138]:
```suggestion
-   Avoid global variables, use `local VARIABLE` whenever possible
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,34 +2,31 @@
 
 ## General
 
+-   Make sure to have a working local development environment of [ddev](https://github.com/ddev/ddev).
 -   Make sure to have a working local development environment, e.g. ddev
```
Comment (Morgy93) [inline CONTRIBUTING.md:6]:
```suggestion
-   Make sure to have a working local development environment of [DDEV](https://github.com/ddev/ddev).
```

**Review by Morgy93 — CHANGES_REQUESTED** [review]

**Comment by dermatz** [comment]
@Morgy93 any news about this PR?

**Comment by Morgy93** [comment]
> @Morgy93 any news about this PR?

There's one open conversation that needs to be resolved:
https://github.com/dermatz/ddev-woodoo-buildtools-magento/pull/49#discussion_r1728435592

**Pull request merged**